src/deluge/gui/l10n/seven_segment.cpp

```diff
@@ -361,17 +361,10 @@ PLACE_SDRAM_DATA Language seven_segment{
 
```

Comment:
re-grouped automation view strings together at the bottom to make it them easier to find and adjust if necessary.